src/js/model/state.js

```diff
@@ -0,0 +1,22 @@
+export default class State {
+  constructor(player, count) {
+    this.player = player;
+    this.count = count;
+  }
+
+  get player() {
+    return this._player;
+  }
+
+  set player(value) {
+    return this._player = value;
+  }
+
+  get count() {
+    return this._count;
+  }
+
+  set count(value) {
+    return this._count = value;
+  }
+}
```

Comment:
```suggestion
export const State = {
  player: null,
  count: null,
}
```
그냥 이렇게 관리하는거랑 다를게 있을까요?

Reply:
class를 사용하는 이유는 재상용성이고 getter, setter를 사용하는 이유는 방어적인 코드 작성을 위함인데 여기서는 불필요하다고 생각이 되네요 제거 해도 될거같아요 !